### ISSUE TITLE
API docs error

### ISSUE THREAD

**Issue by davidchase** [cujojs/most#88]
So i know there are talks about depreciating `fromEventWhere` #86 which i am +1, if you can have a smaller api that is compose-able  rather than all inclusive that seems better especially if `fromEventWhere` seemed to be a temp solution.

Nonetheless in the API you have
```js
var form = document.querySelector('.container');
most.fromEventWhere(function(e) {
        return e.matches('.toggle-button') && e.preventDefault();
    }, 'click', container)
    .forEach(doSomething);
```
I believe you mean the `var form` to be `var container`
and i do not believe it's possible to do `element.matches` since the argument `e` is event correct?

**Comment by briancavalier** [comment]
@davidchase Heh, I think there are *two* typos in that code!  It should use `e.target.matches`, to avoid capturing, and `form` and `container` are supposed to be the same variable.

Thanks, want to send a PR?

**Comment by davidchase** [comment]
@briancavalier sure thing, what are your thoughts on showing an example how the same thing can be done with `most.fromEvent(...).filter(e => e.target.matches(...))` to avoid capturing in the API docs or is it unnecessary right now since you haven't depreciated `fromEventWhere`

**Comment by briancavalier** [comment]
Sure, go ahead and include that in the PR and maybe that'll help in making a final decision on deprecating `fromEventWhere`.

**Comment by davidchase** [comment]
added my PR #89

**Comment by briancavalier** [comment]
Closed by #89
